src/plugins/intel_npu/src/plugin/npuw/compiled_model.cpp

```diff
@@ -387,6 +394,23 @@ ov::npuw::CompiledModel::CompiledModel(const std::shared_ptr<ov::Model>& model,
     reset_io();
 }
 
+void ov::npuw::CompiledModel::fill_weights_bank(const std::size_t idx) {
+    LOG_VERB("Filling weights bank for Subgraph[" << idx << "]...");
+    LOG_BLOCK();
+    auto& comp_model_desc = m_compiled_submodels[idx];
+
+    for (std::size_t cidx = 0u; cidx < comp_model_desc.closure.size(); cidx++) {
+        comp_model_desc.closure[cidx] = m_weights_bank->get(comp_model_desc.closure[cidx]);
+        if ("YES" == m_cfg.getString<::intel_npu::NPUW_CWAI>()) {
+            comp_model_desc.update_required[cidx] = true;
```

Comment:
Actually update (speaking of the execution time update) is NOT required in this case.

Reply:
Done